src/map_io/map_players_messages_packet.cc

```diff
@@ -119,27 +127,23 @@ void MapPlayersMessagesPacket::read(FileSystem& fs,
 						assert(mol.is_object_loaded(mo));
 						serial = mo.serial();
 					}
-					// Compatibility code needed for map loading.
```

Comment:
we normally try to keep map compatibility for the longest time possible. so unless it is proven that a map from build 18 won't load anyway we should keep this.